ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_portugal_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTA0NTIwMDAwMy4xNzQ2MzE1ODc2In0sImFub25JZCI6IjJjZDNkZTA2YzUxODQ2OTcifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnoucHQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2dsZS9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MSIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjUwMTExODU1Njc4OTAwNjciLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55e8957e275a <unknown>
#1 0x55e8952954b0 <unknown>
#2 0x55e89527bd86 <unknown>
#3 0x55e89527a69a <unknown>
#4 0x55e89527adff <unknown>
#5 0x55e89527ad54 <unknown>
#6 0x55e89529969f <unknown>
#7 0x55e89533e92a <unknown>
#8 0x55e89530c592 <unknown>
#9 0x55e8953325e0 <unknown>
#10 0x55e89530c363 <unknown>
#11 0x55e8952d8d63 <unknown>
#12 0x55e8952d99c1 <unknown>
#13 0x55e8957a7a6b <unknown>
#14 0x55e8957ab951 <unknown>
#15 0x55e89578fb62 <unknown>
#16 0x55e8957ac4c4 <unknown>
#17 0x55e89577413f <unknown>
#18 0x55e8957d06f8 <unknown>
#19 0x55e8957d08d6 <unknown>
#20 0x55e8957e15a6 <unknown>
#21 0x7f947089caa4 <unknown>
#22 0x7f9470929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_portugal_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAxNjcxIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJwdF9QVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LnB0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGUvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjEifSwiZmxhZ3MiOnsicGFnZVZpZXciOnRydWV9LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEwNDUyMDAwMDMuMTc0NjMxNTg3NiJ9LCJhbm9uSWQiOiIyY2QzZGUwNmM1MTg0Njk3In0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6Mn0sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiIxNjcxIiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyIxNjcxIl19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiOTE5ODk4NTU1MzY3MTk1In0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55e8957e275a <unknown>
#1 0x55e8952954b0 <unknown>
#2 0x55e89527bd86 <unknown>
#3 0x55e89527a69a <unknown>
#4 0x55e89527adff <unknown>
#5 0x55e89527ad54 <unknown>
#6 0x55e89529969f <unknown>
#7 0x55e89533e92a <unknown>
#8 0x55e89530c592 <unknown>
#9 0x55e8953325e0 <unknown>
#10 0x55e89530c363 <unknown>
#11 0x55e8952d8d63 <unknown>
#12 0x55e8952d99c1 <unknown>
#13 0x55e8957a7a6b <unknown>
#14 0x55e8957ab951 <unknown>
#15 0x55e89578fb62 <unknown>
#16 0x55e8957ac4c4 <unknown>
#17 0x55e89577413f <unknown>
#18 0x55e8957d06f8 <unknown>
#19 0x55e8957d08d6 <unknown>
#20 0x55e8957e15a6 <unknown>
#21 0x7f947089caa4 <unknown>
#22 0x7f9470929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_portugal_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEwNDUyMDAwMDMuMTc0NjMxNTg3NiJ9LCJhbm9uSWQiOiIyY2QzZGUwNmM1MTg0Njk3In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LnB0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGUvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjEiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoyfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIxMTc5MDAzOTEyODc2MDg1NCIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55e8957e275a <unknown>
#1 0x55e8952954b0 <unknown>
#2 0x55e89527bd86 <unknown>
#3 0x55e89527a69a <unknown>
#4 0x55e89527adff <unknown>
#5 0x55e89527ad54 <unknown>
#6 0x55e89529969f <unknown>
#7 0x55e89533e92a <unknown>
#8 0x55e89530c592 <unknown>
#9 0x55e8953325e0 <unknown>
#10 0x55e89530c363 <unknown>
#11 0x55e8952d8d63 <unknown>
#12 0x55e8952d99c1 <unknown>
#13 0x55e8957a7a6b <unknown>
#14 0x55e8957ab951 <unknown>
#15 0x55e89578fb62 <unknown>
#16 0x55e8957ac4c4 <unknown>
#17 0x55e89577413f <unknown>
#18 0x55e8957d06f8 <unknown>
#19 0x55e8957d08d6 <unknown>
#20 0x55e8957e15a6 <unknown>
#21 0x7f947089caa4 <unknown>
#22 0x7f9470929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_portugal_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LnB0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGUvb3ZlcnZpZXcuaHRtbCIsInZlaGljbGVDb2RlIjoidjE2Ny1mbCIsInZlaGljbGVCTSI6IjE2NzEiLCJ2ZWhpY2xlUElIaXN0b3J5IjoiMTY3MSIsInRpbWVzdGFtcCI6MTc0NjMxNTg4MTk3MiwiZXhwaXJhdGlvbiI6MTIwfX19LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjEwNDUyMDAwMDMuMTc0NjMxNTg3NiJ9LCJhbm9uSWQiOiIyY2QzZGUwNmM1MTg0Njk3In0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LnB0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGUvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjEiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJfdG9vbHNFdmVudExpbmtJZCI6Ijc2NjA2MzI1MTg5NTI5MzkiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55e8957e275a <unknown>
#1 0x55e8952954b0 <unknown>
#2 0x55e89527bd86 <unknown>
#3 0x55e89527a69a <unknown>
#4 0x55e89527adff <unknown>
#5 0x55e89527ad54 <unknown>
#6 0x55e89529969f <unknown>
#7 0x55e89533e92a <unknown>
#8 0x55e89530c592 <unknown>
#9 0x55e8953325e0 <unknown>
#10 0x55e89530c363 <unknown>
#11 0x55e8952d8d63 <unknown>
#12 0x55e8952d99c1 <unknown>
#13 0x55e8957a7a6b <unknown>
#14 0x55e8957ab951 <unknown>
#15 0x55e89578fb62 <unknown>
#16 0x55e8957ac4c4 <unknown>
#17 0x55e89577413f <unknown>
#18 0x55e8957d06f8 <unknown>
#19 0x55e8957d08d6 <unknown>
#20 0x55e8957e15a6 <unknown>
#21 0x7f947089caa4 <unknown>
#22 0x7f9470929c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'